src/djangosaml2_spid/identity_providers.py

```diff
@@ -0,0 +1,17 @@
+# source: https://registry.spid.gov.it/identity-providers
+
+import urllib.request
+import json
+
+identity_providers_url = 'https://registry.spid.gov.it/assets/data/idp.json'
+
+
+def download_identity_providers():
```

Comment:
is it a general tuility or you want to use it somewhere for real?

Reply:
This is still a work in progress, the idea is to make it download the IdP on user intervention and put all the metadata in the `local` folder, just to not do it by hand. Maybe we can discuss a better strategy?

Reply:
I wrote a first version of the utility and added instruction to the README.md, check it out! 😄